chef-config/lib/chef-config/config.rb

```diff
@@ -320,6 +344,21 @@ def self.path_accessible?(path)
       default(:enabled) { ChefConfig::Config.local_mode }
       default :host, "localhost"
       default :port, 8889.upto(9999) # Will try ports from 8889-9999 until one works
+
+      # When set to a String, Chef Zero disables multitenant support.  This is
+      # what you want when using Chef Zero to serve a single Chef Repo. Setting
+      # this to `false` enables multi-tenant.
```

Comment:
Maybe it could work by accident, but the effect of this is to prepend requests with `/organizations/$single_org_option`, as is done here: https://github.com/chef/chef-zero/blob/57f2536f25b06930dbbf3560dfc6e2b0c644af03/lib/chef_zero/server.rb#L624 It's also passed in to some of the data store stuff, I didn't look deeper there but you could start here if you wanted to look into it further: https://github.com/chef/chef-zero/blob/57f2536f25b06930dbbf3560dfc6e2b0c644af03/lib/chef_zero/server.rb#L181-L187